handbook/company/communications.md

```diff
@@ -176,6 +176,9 @@ We use these prefixes to organize the Fleet Slack:
 - We configure our [working hours in Slack](https://slack.com/help/articles/360025054173-Set-up-Slack-for-work-hours-) to make sure everyone knows when they can get in touch with others.
 - In consideration of our team, Fleet avoids using global tags in channels (i.e. @here, @channel, etc.) (What about polls? Good question, Fleeties are asked to post their poll in the channel and @mention the teammates they would like to hear from.)
 
+### Create an issue from a Slack thread
```

Comment:
```suggestion
### Create a GitHub issue from a Slack thread
```

Reply:
@alexmitchelliii and @dherder This looks great. LMK what you think about the above. Either way, good to merge